src/cryptoadvance/specter/templates/wallet/receive/wallet_receive.jinja

```diff
@@ -217,6 +225,11 @@
 				btn.innerText = 'Hide past addresses';
 			}
 		}
+		function toggleKeyDisplay() {
+			let slip132Checkbox = document.getElementById('use_slip_132_pdf');
+			let xpubs = document.getElementsByClassName('slip132_xpub');
+			let zpubs = document.getElementsByClassName('normal_xpub');
+		}
```

Comment:
What is this for? Looks like it doesn't do anything...

Reply:
Done, commented out the selectors and left a minimal detail on why they are not completely removed.